content/spin/v2/http-trigger.md

```diff
@@ -78,6 +79,10 @@ route = "/users/..."
 component = "user-manager"
 ```
 
+A single-segment wildcard uses the syntax `:name`, where `name` is a name that identifies the wildcard. Such a wildcard will match only a single segment of a path, and allows further matching on segments beyond it. For example, `/users/:userid/edit` matches `/users/1/edit` and `/users/alice/edit`, but does not match `/users`, `/users/1`, or `/users/1/edit/cart`.
```

Comment:
remove if suggestion accepted (the suggestion moves this text to be part of the existing wildcard content)